.gitignore

```diff
@@ -137,3 +138,6 @@ build
 /site/Gemfile.lock
 /site/.sass-cache
 /site/_site
+
+# Pytest Cache
+**/.pytest_cache
```

Comment:
I think we still will need a newline at the end of the file.

Reply:
Done